spec/abstractops.html

```diff
@@ -1785,22 +1810,19 @@ <h1>
     </dl>
     <emu-alg>
       1. Set _options_ to ? GetOptionsObject(_options_).
-      1. Let _smallestUnit_ be ? GetTemporalUnit(_options_, *"smallestUnit"*, _unitGroup_, _fallbackSmallestUnit_).
-      1. If _disallowedUnits_ contains _smallestUnit_, throw a *RangeError* exception.
-      1. Let _defaultLargestUnit_ be ! LargerOfTwoTemporalUnits(_smallestLargestDefaultUnit_, _smallestUnit_).
       1. Let _largestUnit_ be ? GetTemporalUnit(_options_, *"largestUnit"*, _unitGroup_, *"auto"*).
```

Comment:
This insertion of ToTemporalRounding operations to read "rounding…" properties in between reading "largestUnit" and "smallestUnit" seems non-obvious and therefore brittle.
```suggestion
      1. NOTE: The following steps read options and perform independent validation in alphabetical order (ToTemporalRoundingIncrement reads *"roundingIncrement"* and ToTemporalRoundingMode reads *"roundingMode"*).
      1. Let _largestUnit_ be ? GetTemporalUnit(_options_, *"largestUnit"*, _unitGroup_, *"auto"*).
```
(and similarly in other algorithms for which we want to ensure alphabetical property reads, including all changes in this PR and also the previously-updated ToTemporalPartialDurationRecord)